frontend/.storybook/preview.js

```diff
@@ -1,7 +1,7 @@
 import { theme as V5Theme } from '@/v5/ui/themes/theme';
 import { theme as V5ViewerTheme } from '@/v5/ui/routes/viewer/theme';
```

Comment:
Leaving a sequence is no longer resetting the transform
![24-01-22 01-25-38](https://github.com/3drepo/3drepo.io/assets/11945337/9d136522-b169-44c3-b9cf-907a574a14f5)